templates/data/postgresql/managed.html

```diff
@@ -0,0 +1,465 @@
+    <div class="col">
+      <h2>PostgreSQL resources</h2>
+    </div>
+    <div class="col">
+      <ul class="p-list--divided">
+        <li class="p-list__item">
+          <h3 class="p-heading--5 u-no-margin--bottom">
+            <a href="https://ubuntu.com/engage/database-cloud-migration">A guide to database cloud migration</a>
+          </h3>
+          <p>Gain insights on how to succeed in migrating your most valuable assets, your data, to the cloud.</p>
+        </li>
+        <li class="p-list__item">
+          <h3 class="p-heading--5 u-no-margin--bottom">
+            <a href="https://ubuntu.com/engage/data-backups">A guide to data backups</a>
+          </h3>
+          <p>Build a comprehensive backup strategy to meet your requirements in terms of data protection and recovery times.</p>
+        </li>
+        <li class="p-list__item">
+          <h3 class="p-heading--5 u-no-margin--bottom">
+            <a href="https://ubuntu.com/engage/postgresql-mysql">PostgreSQL vs MySQL webinar</a>
+          </h3>
+          <p>Dive into the differences between the two most popular open-source databases and what those differences mean from a performance, operability and community engagement perspective.</p>
+        </li>
+        <li class="p-list__item">
+          <h3 class="p-heading--5 u-no-margin--bottom">
+            <a href="https://ubuntu.com/engage/postgres-extensions">How many use cases can you cover with PostgreSQL extensions?</a>
+          </h3>
+          <p>Examine the use cases you can meet with just PostgreSQL.</p>
+        </li>
+        <li class="p-list__item">
+          <h3 class="p-heading--5 u-no-margin--bottom">
+            <a href="https://ubuntu.com/engage/postgresql-replication">A deep dive into PostgreSQL replication and automatic failover techniques</a>
+          </h3>
+          <p>Delve into advanced techniques for implementing robust replication and automatic failovers in PostgreSQL.</p>
+        </li>
+      </ul>
+    </div>
+  </div>
+</section>
+<div class="u-fixed-width">
+  <hr class="p-rule" />
+</div>
+<section class="p-strip is-deep">
+  <div class="u-fixed-width">
+    <h2><a href="/data/postgresql#get-in-touch">Discuss your needs with our team&nbsp;&rsaquo;</a></h2>
+  </div>
+</section>
+<section class="p-section">
+  <div class="u-fixed-width">
+    <hr class="p-rule" />
+  </div>
+  <div class="row">
+    <div class="col-6 col-start-large-7">
+      <p><small class="u-text--muted">“PostgreSQL” is a trademark or registered trademark of <a href="https://www.postgres.ca/">PGCA</a>. Other trademarks are property of their respective owners. Canonical’s PostgreSQL offerings are not sponsored, endorsed, or affiliated with <a href="https://www.postgres.ca/">PGCA</a>.</small></p>
+    </div>
+  </div>
+</section>
+
+<script defer src="{{ versioned_static('js/modals.js') }}"></script>
+
+{% with %}
+  {% set returnURL = "https://canonical.com/data/postgresql/managed#success" %}
```

Comment:
For c.com links pls remove site name and use only "/data/postgresql/managed#success"